tests/ui/coroutine/polymorphize-args.rs

```diff
@@ -1,4 +1,3 @@
-//@ compile-flags: -Zpolymorphize=on
```

Comment:
pls remove this test, it's no longer needed if we're not polymorphizing anything.